packages/services/api/src/modules/schema/providers/registry-checks.ts

```diff
@@ -139,7 +133,6 @@ export class RegistryChecks {
           errorsBySource: {
             graphql: validationErrors.filter(isGraphQLValidationError),
             composition: validationErrors.filter(isCompositionValidationError),
-            policy: validationErrors.filter(isPolicyValidationError),
```

Comment:
Compose and validate can never yield policy errors, it does not make sense to add them here.